### ISSUE TITLE
Od dziś oferty przez API mogą wystawiać i edytować tylko użytkownicy z kontem firmowym / From today only users with company account can list and edit offers via API

### ISSUE THREAD

**Issue by PrzemyslawLukanowski** [allegro/allegro-api#5935]
Zgodnie z naszą [wcześniejszą zapowiedzią](https://developer.allegro.pl/news/od-12-lipca-2022-oferty-przez-api-beda-mogly-wystawiac-i-edytowac-tylko-konta-firmowe-aMGXmOdyXhq) wyłączyliśmy dziś możliwość wystawiania i edycji ofert przez API dla użytkowników z kontem zwykłym.

Aby dalej wystawiać oferty na Allegro, [zmień status swojego konta](https://allegro.pl/pomoc/dla-sprzedajacych/dane-i-ustawienia-konta/kiedy-i-jak-moge-zmienic-status-konta-rKxBgnr6ksx) na firmowe.

-----

According to our [previous announcement](https://developer.allegro.pl/news/od-12-lipca-2022-oferty-przez-api-beda-mogly-wystawiac-i-edytowac-tylko-konta-firmowe-aMGXmOdyXhq), as of today, we have disabled the possibility to create and edit offers via API for users with a regular account.

[Change the status](https://allegro.pl/pomoc/dla-sprzedajacych/dane-i-ustawienia-konta/kiedy-i-jak-moge-zmienic-status-konta-rKxBgnr6ksx) of your account to a company to continue listing offers on Allegro.

**Comment by PhotoSoft** [comment]
Czy rozesłaliście mailing do użytkowników, których dotyczy zmiana? Bo już mam maile w stylu "dlaczego program przestał działać" 😡

**Comment by PrzemyslawLukanowski** [comment]
Tak, mailing był wysyłany 12 lipca o 16:00, a jego tytuł to "Sprzedajesz na koncie zwykłym? Od 27 lipca nie wystawisz i nie edytujesz ofert przez API oraz z pliku".

**Comment by PhotoSoft** [comment]
Obawiam się, że mało który użytkownik zwykłego konta wie co to API. Gdybyście napisali, że nie będzie można obsługiwać ofert poprzez zewnętrzne oprogramowanie to byłoby to bardziej zrozumiałe dla nich, a tak będą nas zasypywać pytaniami i oczywiście nas obwiniać...

**Comment by PrzemyslawLukanowski** [comment]
Dziękuję za tę sugestię, przekazałem ją do zespołu odpowiedzialnego za komunikację do sprzedających.
